### DISCUSSION TITLE
embed values from codecells in markdown cells?

### DISCUSSION THREAD

**Discussion by actsasgeek** [quarto-dev/quarto-cli#1804] [Q&A] [answered]
I'm not sure if this is a feature of R or Rmd or how qmd differs. Basically, I'm looking to see the best way to implement this feature from jupyter book:

https://jupyterbook.org/en/stable/content/executable/output-insert.html#glue-gluing

Basically, you would have a value assigned to a variable in a code cell and then be able to reference it in a markdown cell. This is a pretty slick way of being able to reference values that can change (and so the discussion should change). You can also do nifty things like sparklines.

**Comment by jjallaire** [comment, marked as answer]
The best way to do this right now is to generate the Markdown dynamically in a code cell. For example see: https://quarto.org/docs/computations/execution-options.html#inline-code

We will likely at some point implement a more compact/elegant syntax for this.
